backend/src/test/java/com/team4/backend/service/UserServiceTest.java

```diff
@@ -53,8 +56,11 @@ void login() {
         Mono<String> returnedToken2 = userService.login(authRequestDto2);
 
         //ASSERT
-        returnedToken1.subscribe(subToken -> assertEquals(token, subToken));
-        returnedToken2.subscribe(Assertions::assertNull);
+        StepVerifier.create(returnedToken1)
+                .consumeNextWith(subToken -> assertEquals(token,subToken))
```

Comment:
Please use other name than subToken as its confusing to the nature of the token

Reply:
https://github.com/0xD34DC0DE/ProjetIntegreTeam4/pull/3/commits/a6686770d948846ef5965558e2a222ee30b1f7e4